libraries/AP_InertialSensor/AP_InertialSensor_MPU9250.cpp

```diff
@@ -438,7 +438,7 @@ bool AP_InertialSensor_MPU9250::_accumulate(uint8_t *samples, uint8_t n_samples)
             _fifo_reset();
             return false;
         }
-        float temp = t2/340 + 36.53;
+        float temp = static_cast<float>(t2)/340 + 36.53;
```

Comment:
@OXINARF I'd agree with that. I find the C++ one to be very verbose for the purpose.